ARMTemplates/blobaadauthsink-sample/azuredeploy.json

```diff
@@ -239,7 +240,8 @@
               }
             ],
             "typeProperties": {
-              "tableName": "BasicDataSet_v0.Event_v0"
+              "tableName": "BasicDataSet_v0.Event_v0",
+              "predicate": "LastModifiedDateTime >= 2016-10-18T08:50:27Z AND LastModifiedDateTime <= 2018-12-18T10:00:00Z"			  
```

Comment:
Is LastModified a good default? We have used CreateDateTime elsewhere

Reply:
User has to specify either one of these predicates since we are mandating it. I added different filters to provide an example for different filters. 

If the goal is to keep it consistent, I can use the same filters as in other files.

Reply:
Made it consistent. Used default predicates for messages, events and contacts